Procfile

```diff
@@ -1,3 +1,3 @@
-publishing-api-worker: bundle exec sidekiq -C ./config/sidekiq/publishing_api.yml
-google-analytics-worker: bundle exec sidekiq -C ./config/sidekiq/google_analytics.yml
+publishing-api-worker: bundle exec sidekiq -C ./config/sidekiq/publishing_api.yml --tag publishing-api-worker-process
```

Comment:
This might be a little too open to confusion, seen as there is also an application called the publishing-api. Maybe `content-audit-tool/publishing-api-worker` would be more specific?